src/main/java/com/iridium/iridiumskyblock/commands/DepositCommand.java

```diff
@@ -47,7 +47,13 @@ public void execute(CommandSender sender, String[] args) {
         if (island.isPresent()) {
             Optional<BankItem> bankItem = IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getName().equalsIgnoreCase(args[1])).findFirst();
             if (bankItem.isPresent()) {
-                double amount = bankItem.get().deposit(player, Double.parseDouble(args[2]));
+                double amount;
+                try {
+                    amount = bankItem.get().deposit(player, Double.parseDouble(args[2]));
+                } catch (NumberFormatException e) {
+                    player.sendMessage(IridiumSkyblock.getInstance().getMessages().notANumber.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix));
+                    return;
+                }
                 if (amount > 0) {
                     IslandLog islandLog = new IslandLog(island.get(), LogAction.BANK_DEPOSIT, user, null, amount, bankItem.get().getName());
                     IridiumSkyblock.getInstance().getDatabaseManager().getIslandLogTableManager().addEntry(islandLog);
```

Comment:
Wrong branch?

Reply:
Tried to slip this into the pull request. The branch is called "fixes", so no, not the wrong branch. Wrong pull request? Maybe xD

Reply:
Renamed pull request to dodge rant about how I should be using a new branch and pull request.

Reply:
Yes, you really should